### DISCUSSION TITLE
Sort by tag / label?

### DISCUSSION THREAD

**Discussion by philipeliot** [FreshRSS/FreshRSS#3551] [Q&A]
Hello,
I've been using RSSOwl (desktop rss reader) for some time but have just installed FreshRSS on my server and would like to replace RSSOwl with FreshRSS.

But one small feature on RSSOwl is very important to the way I use it.

EG: The 'New York Times' feed.
When I import the NYT feed into RSSOwl, it automatically creates a sortable column with each articles' author name, and category.

So, from the feed: https://rss.nytimes.com/services/xml/rss/nyt/HomePage.xml
It creates sortable columns from:
'dc:creator' (author name), and 'category' tags/labels, in addition to the source, title and date, of course.

![Screenshot (2384)](https://user-images.githubusercontent.com/28823277/111918639-bca74500-8a8e-11eb-9515-58e72452e487.png)

AS you can see, I was able to sort the entries by author name in this pic (alphabetically)

But FreshRSS simply adds the author name without the option to sort:

https://nimb.ws/zR3cSL

The author name is there, but I can't order them alphabetically. Also, the category info isn't present.

I see that FreshRSS has a tagging/labelling feature, but I haven't been able to figure out if it is adaptable to my use case.
Author names and categories are multiple and it would be impossible to set up a separate search criteria for each, but that's the only sorting option I can see.

Can anyone help me understand if FreshRSS is able to do this?
Thanks.

**Comment by Alkarex** [comment]
Hello,
In FreshRSS, you can filter by e.g. author or tag, but not sort by author or tag at the moment. However, there are multiple (desktop) clients compatible with FreshRSS, some of which have additional features.
The labels in FreshRSS are current manual (you can tag articles one at a time), but there is an auto-labelling features coming soon, based on automatic search queries.

**Reply by MundoMedia** [reply]
Thanks @Alkarex 

I guess I'll have to find something else. It's a shame because I like the look of FreshRSS and in most respects I thought it would do the job but the ability to capture, and sort, by labels/tags is an absolute necessity -- in fact, I can't figure out how people use a reader without this functionality!

Thanks for your help and taking time out to reply.

**Comment by superkeyor** [comment]
Sorry to revive this old thread. Given that we now have the LLM Classifier extension, would it make more sense to add a sorting-by-tag functionality?
